recipes/libcap/all/conanfile.py

```diff
@@ -23,6 +25,8 @@ class LibcapConan(ConanFile):
         "fPIC": True,
         "psx_syscals": False,
     }
+    exports_sources = "patches/**/**"
```

Comment:
Does `patches/**` suffice for multiple levels?

Reply:
Actually it is. I don't remember why i was trying to modify it. Thanks.